recipe/meta.yaml

```diff
@@ -1,16 +1,13 @@
 {% set name = "jedi" %}
-{% set version = "0.9.0" %}
-{% set hash_type = "sha256" %}
-{% set hash = "3b4c19fba31bdead9ab7350fb9fa7c914c59b0a807dcdd5c00a05feb85491d31" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type }}: {{ hash }}
+  git_url: https://github.com/davidhalter/jedi.git
+  git_tag: v{{ version }}
```

Comment:
You want version `0.10.0`, right? Then you can use GitHub tarball like I suggested above. Here is a copy-and-paste version of the changes we require:

```yaml
    fn: jedu-v{{ version }}.tar.gz
    url: https://github.com/davidhalter/jedi/archive/v{{ version }}.tar.gz
    sha256: d6a7344df9c80562c3f62199278004ccc7c5889be9f1a6aa5abde117ec085123
```